src/QuiqChatClient.js

```diff
@@ -63,11 +70,19 @@ class QuiqChatClient {
 
   onError = (callback: (error: ?ApiError) => void): QuiqChatClient => {
     this.callbacks.onError = callback;
+    registerCallbacks({onError: callback});
```

Comment:
I think I get what you're saying but I'd like to walk through this with you further when we get a chance.  Going to bounce until then